addons/account/views/account_move_views.xml

```diff
@@ -706,20 +706,29 @@
                                 <label for="journal_id"
                                        groups="account.group_account_readonly"
                                        invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
-                                <div name="journal_div" class="d-flex" invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'">
+                                <div name="journal_div"
+                                     class="d-flex"
+                                     groups="account.group_account_readonly"
+                                     invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'">
                                     <field name="journal_id"
-                                        groups="account.group_account_readonly"
-                                        options="{'no_create': True}"
-                                        attrs="{'readonly': [('posted_before', '=', True)]}"/>
-                                    <span class="oe_inline o_form_label mx-3 oe_read_only"
-                                        groups="base.group_multi_currency"> in </span>
-                                    <!-- Bigger margin on the left because in edit mode the external link button covers the text -->
-                                    <span class="oe_inline o_form_label mr-3 ml-5 oe_edit_only"
-                                        groups="base.group_multi_currency"> in </span>
+                                           options="{'no_create': True, 'no_open': True}"
+                                           attrs="{'readonly': [('posted_before', '=', True)]}"/>
+                                    <span class="oe_inline o_form_label mx-3"
+                                          groups="base.group_multi_currency"
+                                          attrs="{'invisible': [('move_type', '=', 'entry')]}"> in </span>
```

Comment:
```
    attrs="{'invisible': [('move_type', '=', 'entry')]}"
```
Same question, was this asked/validated by @tsb-odoo ? This is a change in behavior compared to what we had

Reply:
Will remove it, it's unneeded. 👍🏻 
If it's an entry, the whole block is invisible and the next block is visible, and vice-versa.

Reply:
No, wait, the two `invisible` are different. 
![image](https://user-images.githubusercontent.com/1665365/199939006-511e96f0-82fb-4436-956b-eb3694d8d068.png)

This `invisible` also comes from `16.0`
https://github.com/odoo/odoo/blob/16.0/addons/account/views/account_move_views.xml#L843
Will remove it.